src/a-st-ext.adoc

```diff
@@ -250,6 +253,7 @@ those with both bits clear, but may result in lower performance.
 LR/SC can be used to construct lock-free data structures. An example
 using LR/SC to implement a compare-and-swap function is shown in
 <<cas>>. If inlined, compare-and-swap functionality need only take four instructions.
+====
```

Comment:
Sample code should generally be non-normative.  Please post a screenshot of the effect of this change so we can make sure it looks OK.